models/stdp_connection.h

```diff
@@ -68,11 +68,12 @@
 
   FirstVersion: March 2006
   Author: Moritz Helias, Abigail Morrison
+  Adapted by: Philipp Weidel
   SeeAlso: synapsedict, tsodyks_synapse, static_synapse
 */
 
 // C++ includes:
-#include <cmath>
+#include <math.h>
```

Comment:
We were afraid that `copysign` is not available in the c++ header for some supercomputers. Therefore, I changed it to the C header and we tested this for several compilers. Please read the comments above.